### ISSUE TITLE
Hyper-V Replica and Cloud Orchestration 

### ISSUE THREAD

**Issue by fenykepesz** [selagroup/sdp-june2016-cfp#48]
### Hyper-V Replica and Cloud Orchestration

**Type:** Breakout session 

**Speaker:**

**Technical level:** Intermediate

**Abstract:** Our world is becoming increasingly more hostile. Terror, natural disasters and plain human error loom over us all...and more importantly, they loom over our favorite social network sites!

But alas, not all is lost. To feed our appetite for being able to access everything, everywhere and always- the people that operate our critical infrastructures (yes, social networking is critical...kind of...) will employ high availability(HA) and disaster recovery(DR) solutions. 

These topics are hot, but relatively complicated. Implementation is far from simple, and at times it might require special equipment and huge investments. 

Luckily, by joining this session you will obtain insight on Hyper-V Replica, and Cloud Orchestration, a relatively simple solution that may be the answer for your HA and DR needs.

**Prerequisites:** Basic understanding of Hyper-V, HA and DR concepts.

**Comment by goldshtn** [comment]
@erozman Erik, thank you for your submission. Can you please create another issue with your speaker profile? You can find instructions for doing so at the main page.

**Comment by fenykepesz** [comment]
@goldshtn Thank you for considering me, I will create a speaker profile shortly.

**Comment by goldshtn** [comment]
@erozman Erik, I think you forgot to create the speaker profile. Could you please do so?
